denops/ddu/utils.ts

```diff
@@ -5,33 +5,29 @@ export function treePath2Filename(treePath: TreePath) {
   return typeof treePath === "string" ? treePath : treePath.join(pathsep);
 }
 
+export async function printError(
+  denops: Denops,
+  ...messages: unknown[]
+) {
+  const message = messages.map((v) => {
+    if (v instanceof Error) {
+      // NOTE: In Deno, Prefer `Error.stack` because it contains `Error.message`.
+      return `${v.stack ?? v}`;
+    } else if (typeof v === "object") {
+      return JSON.stringify(v);
+    } else {
+      return `${v}`;
+    }
+  }).join("\n");
+  await denops.call("ddu#util#print_error", message);
+}
+
 export async function errorException(
```

Comment:
This function should be replaced with `printError`.

Reply:
Does that mean remove `errorException`?

Reply:
Yes.